.husky/pre-push

```diff
@@ -1,5 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-npm run lerna-lint-fix
 npm run test
```

Comment:
I think we should change this to `npm test -ws`?

Currently, it will run `npm run lerna-test` which will run tests in all packages, examples, and layer-publisher.
If instead we changed it to the above we'd be running the unit tests only in `packages/*` and not in the others.

This is on line with what we discussed in the last conversation about the topic and is captured in #1106. If you want to keep this separate I can open a PR later on.

Aside from where the tests are run only difference between `npm run test` & `npm test -ws` is that npm workspaces still doesn't run in parallel but sequentially. I don't think it's a big deal at this stage, but if you want to run it in parallel you can use `npm t -w packages/commons & npm t -w packages/logger & npm t -w packages/tracer & npm t -w packages/metrics` instead.